# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/mp4_demuxer/TestParser.cpp
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 dom/media/gtest/mp4_demuxer/TestParser.cpp
--- a/dom/media/gtest/mp4_demuxer/TestParser.cpp	Fri Jan 27 19:06:16 2023 +0000
+++ b/dom/media/gtest/mp4_demuxer/TestParser.cpp	Fri Jan 27 21:11:42 2023 +0200
@@ -316,7 +316,7 @@
           << tests[test].mFilename;
       if (tests[test].mHasVideoIndice) {
         for (size_t i = 0; i < indices.Ref()->Length(); i++) {
-          MP4SampleIndex::Indice data;
+          Index::Indice data;
           EXPECT_TRUE(indices.Ref()->GetIndice(i, data))
               << tests[test].mFilename;
           EXPECT_TRUE(data.start_offset <= data.end_offset)
@@ -349,7 +349,7 @@
           metadata.GetTrackIndice(audioInfo->mTrackId);
       EXPECT_TRUE(!!indices.Ref()) << tests[test].mFilename;
       for (size_t i = 0; i < indices.Ref()->Length(); i++) {
-        MP4SampleIndex::Indice data;
+        Index::Indice data;
         EXPECT_TRUE(indices.Ref()->GetIndice(i, data)) << tests[test].mFilename;
         EXPECT_TRUE(data.start_offset <= data.end_offset)
             << tests[test].mFilename;